apps/stats-web/next.config.js

```diff
@@ -13,6 +13,14 @@ try {
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  webpack: (config, { isServer }) => {
```

Comment:
could you please explain why is this needed?

Reply:
@ygrishajev 
now the Next.js bundler is interfering with how thread-stream resolves its worker files. because worker threads used in server-side libraries are not bundled correctly, i excluded it

Reply:
what issue was it causing exactly? throwing some kind of exception? at build time? at run time?

Reply:
i did some research on it and found a solution by changing `next.config.js` 
@ygrishajev

Reply:
i don't mind this in next.config. I just was curious what is it